src/promql.ts

```diff
@@ -60,6 +61,7 @@ export const promql = {
   unless: ({ left, right }: LogicalOpParams) => `${left} unless ${right}`,
 
   rate: ({ expr, interval = '$__rate_interval' }: Rate) => `rate(${expr}[${interval}])`,
+  increase: ({ expr, interval = '$__rate_interval' }: Increase) => `increase(${expr}[${interval}])`,
```

Comment:
rate_interval doesn't really make sense for `increase`, I think a better default value could be `$__range`.
```suggestion
  increase: ({ expr, interval = '$__range' }: Increase) => `increase(${expr}[${interval}])`,
```